### ISSUE TITLE
Revert changes needs work

### ISSUE THREAD

**Issue by gbjbaanb** [aspnet/Docs#6974]
Whilst its OK to say "use migrations remove to remove a migration", it only works if you've not applied it to the DB. I don't think there are many cases where a migration was created and removed without updating the DB, so as there is 'down' code in the migrations to remove from the DB, update this documentation to include the correct steps to add a migration, apply it, and then revert it completely so changes can be made and re-applied (as is the most likely scenario for reverting)

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e56e675b-5964-a75b-7e45-6f6d1a607e65
* Version Independent ID: 78e47218-58a3-f8d9-e091-306126871a67
* Content: [ASP.NET Core MVC with EF Core - Migrations - 4 of 10](https://docs.microsoft.com/en-gb/aspnet/core/data/ef-mvc/migrations?view=aspnetcore-2.1)
* Content Source: [aspnetcore/data/ef-mvc/migrations.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/data/ef-mvc/migrations.md)
* Product: **asp.net-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **tdykstra**

**Comment by Rick-Anderson** [comment]
@gbjbaanb  

>I don't think there are many cases where a migration was created and removed without updating the DB

That's correct.  It would be very rare to create a migration and not update. You'll get good error information if you do that.
